test/shopify-cli/partners_api_test.rb

```diff
@@ -69,5 +69,23 @@ def test_query
       api_stub.expects(:query).with('query', variables: {}).returns('response')
       assert_equal 'response', PartnersAPI.query(@context, 'query')
     end
+
+    def test_authenticate_calls_oauth
+      @oauth_client = Object.new
```

Comment:
Minor: I know that we use `Object.new` elsewhere, but I think using a named double, e.g. `mock('oauth_client')` is better as often results a more useful error message when a test fails.

Reply:
Also, I think this can be a local variable rather than instance?